application/commands/removeValidator.go

```diff
@@ -71,33 +61,42 @@ func RemoveCommand() *cobra.Command {
 			if err != nil {
 				log.Printf("Db is already open: %v", err)
 				log.Printf("sending rpc")
+
 				var err2 error
```

Comment:
Why do we need err2 variable and don't reuse err?

Reply:
might have thought some use of `err`, but can replace err2 with err

Reply:
I believe, we have already used `err` in the log above, so it is safe to reuse.